packages/@aws-cdk/aws-s3-deployment/test/lambda/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#---------------------------------------------------------------------------------------------------
+# exeuctes unit tests
+#
+# prepares a staging directory with the requirements
+set -e
+scriptdir=$(cd $(dirname $0) && pwd)
+
+# prepare staging directory
+staging=$(mktemp -d)
+mkdir -p ${staging}
+cd ${staging}
+
+# copy src and overlay with test
+cp -f ${scriptdir}/../../lib/lambda/* $PWD
+cp -f ${scriptdir}/* $PWD
+
```

Comment:
The previous version of this file had:

https://github.com/aws/aws-cdk/blob/b92ed51c6ff11f8453755b6381a3cf1f12b0fcc1/packages/%40aws-cdk/aws-s3-deployment/lambda/test.sh#L20

Which was also actually used in the creation of the bundle. 

https://github.com/aws/aws-cdk/blob/b92ed51c6ff11f8453755b6381a3cf1f12b0fcc1/packages/%40aws-cdk/aws-s3-deployment/lambda/build.sh#L30

How are these dependencies now included in the bundle created in `lambda-layer-awscli`?

Reply:
The `Dockerfile` that builds the layer installs the CLI which includes both.

The requirements were:

```
awscli==1.16.218
boto3==1.9.208
```

Reply:
Why do you say it includes `boto3`? I see it only declares a dependency on `botocore`:

```console
❯ cat layer/awscli/awscli-1.18.198-py2.7.egg-info/requires.txt                                                                                                                                                                              [13:27:30]
botocore==1.19.38
docutils<0.16,>=0.10
s3transfer<0.4.0,>=0.3.0
PyYAML<5.4,>=3.10
colorama<0.4.4,>=0.2.5
rsa<=4.5.0,>=3.1.2
```

Also currently nothing installs `boto3` in the test environment regardless.

Reply:
I think it's installed as part of the Python SDK that's available on Python Lambdas. Having said that we will need to address this very soon (just opened this issue: #12219)

Reply:
<img width="788" alt="Screen Shot 2020-12-24 at 13 33 20" src="https://user-images.githubusercontent.com/598796/103086194-a6ebc600-45ec-11eb-957e-fa1c6084c529.png">